### DISCUSSION TITLE
Github Desktop Error when pushing/syncing/pulling

### DISCUSSION THREAD

**Discussion by chromonym** [community/community#21810] [New to GitHub]
<p>Hey!</p>
<p>I have made a massive edit of one of my python projects in one of my repositories (<a href="https://github.com/NickatGIHS/pykemon" title="Pykémon, some Python Pokémon Projects" rel="nofollow noopener">Pykémon</a>) via GitHub Desktop. However, when I try to push the commit, the following error is shown:</p>
<pre><code>fatal: unable to access 'https://github.com/NickatGIHS/pykemon.git/': schannel: next
InitializeSecurityContext failed: Unknown error (0x80092012) - The revocation function was
unable to check revocation for the certificate.
</code></pre>
<p>Here is an image of the error:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/31c2a0010f81cbf6a9fcc51dfcfbbb706ddf4baf.png" data-download-href="/uploads/short-url/76cnrTRyWmf6dS9mns6CjDrUo23.png?dl=1" title="github-desktop-please-help.PNG" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/44630816/181066978-5bb14a02-878a-4223-9d40-c88c402af5c2.png" alt="github-desktop-please-help.PNG" data-base62-sha1="76cnrTRyWmf6dS9mns6CjDrUo23" width="690" height="254" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/31c2a0010f81cbf6a9fcc51dfcfbbb706ddf4baf_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">github-desktop-please-help.PNG</span><span class="informations">912×336 12.8 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Does anyone know what this means and how I could fix this?</p>
<p>Thanks in advance,</p>
<p><a class="mention" href="/nickatgihs">@nickatgihs</a> </p>
<p>(By the way, I tried the link in the quotes in the error, and it took me to my repository. It’s a public one so you can have a look if you want)</p>

**Comment by chromonym** [comment, marked as answer]
<p>OK, I fixed it simply by moving to a different internet connection. I hope this issue isn’t reoccuring, though.</p>

**Comment by Qube85BB** [comment]
<p>I’m new to GitHub and am having this exact issue. Can you tell me in more detail how you resolved this problem?</p>
<p>Thanks</p>
